.github/workflows/release-desktop-app.yml

```diff
@@ -169,7 +175,17 @@ jobs:
         with:
           name: affine-linux-x64-builds
           path: ./
-
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: release-yml-build-script
+          path: ./
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Generate Release yml
+        run: |
+          BUILD_TYPE=${{ env.BUILD_TYPE }} RELEASE_VERSION=${{ github.event.inputs.version }} node generate-yml.js
```

Comment:
I think you do not need to set BUILD_TYPE here, which is already available in the workflow globally

Reply:
not sure whether can access build type in generate-yml.js

Reply:
checked & removed